cvat/apps/engine/templates/engine/annotation.html

```diff
@@ -137,7 +137,8 @@
                 <polygon points="1,1 1,80 25,60 25,80 50,60 50,80 100,40 50,0 50,20 25,0 25,20" transform="scale(0.4)"/>
             </svg>
             <input type = "range" id = "playerProgress"/>
-        </div>  <!-- END of PLAYER PANEL -->
+        
```

Comment:
I don't think that we need the change.

Reply:
@nmanovic Correct.  This is residue from when I was working on my CF app and directly inserted my script code into this file at this location.  When I cleaned it up and saved, it marked it as a change.